core/src/main/java/org/axonframework/queryhandling/SubscriptionQueryUpdateMessage.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) 2010-2018. Axon Framework
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.axonframework.queryhandling;
+
+import org.axonframework.messaging.Message;
+
+import java.util.Map;
+
+/**
+ * Message which holds incremental update of an subscription query.
+ *
+ * @param <U> type of incremental update
+ * @author Milan Savic
+ * @since 3.3
+ */
+public interface SubscriptionQueryUpdateMessage<U> extends Message<U> {
```

Comment:
Can't this guy extend from the `QueryResponseMessage` interface?
Think that might make a little more sense from a class hierarchy perspective.

Reply:
Definitely, just didn't cross my mind, thanks for the tip ;)

Reply:
No prob! :)